script.js

```diff
@@ -1,43 +1,48 @@
 class Books {
-    constructor(title, author) {
-        this.title = title;
-        this.author = author;
-        this.BookId = this.BookId;
-    }
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
 const bookArchive = [];
-const form  = document.querySelector('.form');
+const form = document.querySelector('.form');
```

Comment:
- [ ] [REQUIRED] Good job so far! Please It should have a function that `removes` a book from the collection (hint: You should use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method) 🙏🏼 
- `[HINT]`
```javascript
const books = [];

const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```
#
![image](https://user-images.githubusercontent.com/63915024/212914368-26ccf88d-b5f4-483b-9b1c-02945e80ae8b.png)
#